data/os/Windows.yaml

```diff
@@ -27,19 +27,13 @@ windows:
   ext_pkg_src: "https://roninpuppetassets.blob.core.windows.net/binaries"
   ext_gpg_src: "https://roninpuppetassets.blob.core.windows.net/binaries/gpg"
 
-  # S3 resources
```

Comment:
@markcor Should we remove this section yet?

Reply:
Let's leave these in place until the moonshots are retired.

Reply:
My bad. Because this is the cloud_windows branch, let;s go ahead and remove those.